<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>busqueda</class>
 <widget class="QDialog" name="busqueda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Resultados de busqueda</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de tarjeta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>PIN de seguridad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Monto de tarjeta</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del cliente</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID cliente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../iconos/close_delete.png</normaloff>../iconos/close_delete.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="modificar_tarjeta">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../iconos/File_Edit.png</normaloff>../iconos/File_Edit.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ver historial</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../iconos/search_button_green_32.png</normaloff>../iconos/search_button_green_32.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultado_numero_tarjeta">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>341</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultado_pin_tarjeta">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>341</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultado_monto_tarjeta">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>341</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultado_nombre_cliente">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>341</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultado_id_cliente">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>341</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
